test/integration/etcd/data.go

```diff
@@ -241,6 +241,13 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		},
```

Comment:
likewise, seems like "`integration etcd data`" ought to be squashed into "`add integration test`"

Reply:
the integration test commit adds tests just for this feature, the `integration etcd data`  is data that is used for another sets of tests, since usually those are different reviewers, I prefer to keep it in different commits